client/modules/Cars/Cars.js

```diff
@@ -129,13 +129,6 @@ export class Car extends Component {
             </form>
```

Comment:
can we make sure that the indentation is consistent. there should be a code formatter tool built into vs code, i think it is ALT + SHIFT + F... either /form is out of place here or <Row is